src/applications/mhv-medical-records/components/CareSummaries/AdmissionAndDischargeDetails.jsx

```diff
@@ -71,7 +71,7 @@ const AdmissionAndDischargeDetails = props => {
     const content = `\n
 ${crisisLineHeader}\n\n
 ${record.name}\n
-${formatName(user.userFullName)}\n
+${formatNameFirstLast(user.userFullName)}\n
```

Comment:
I had not considered that this might include the name of the user themselves. We should verify this with UCD. I suspect that this should remain as is. If we are supposed to do this, then we should also include it in the PDF as well: src/platform/mhv/util/helpers.js (generatePdfScaffold)

Reply:
UCD informed me that they want to see this done with the username too.